Amazon Bedrock Client for Mac/Managers/SettingManager.swift

```diff
@@ -20,20 +20,26 @@ class SettingManager: ObservableObject {
     private let monitoringQueue = DispatchQueue(
         label: "com.amazonbedrock.fileMonitoring", attributes: .concurrent)
     
+    @AppStorage("checkForUpdates") var checkForUpdates: Bool = true
```

Comment:
Here I'm using the `AppStorage` decorator rather than setting manually the `UserDefaults`. The internal implementation is the same, just that this decorator abstract that.